.gitignore

```diff
@@ -115,3 +115,4 @@ env/
 ### Configuration files ###
 .env
 .vscode/settings.json
+config.py
```

Comment:
Setup instructions should probably be mentioned in the README file

Reply:
You still need to update the README. At the moment you are expecting developers to work out for themselves that config.py is required and what it should look like. 

Slightly nicer than an ignored config.py file would be to use environment variables. Add them to the .env.template and your .env file, and then access them in code with os.environ.get, so all your python can be committed. 

Either way, you need to make sure that someone can clone and run the application by just following the README.